src/vs/platform/terminal/common/terminal.ts

```diff
@@ -388,10 +389,9 @@ export interface IShellLaunchConfig {
 	isExtensionOwnedTerminal?: boolean;
 
 	/**
-	 * The codicon ID to use for this terminal. If not specified it will use the default fallback
-	 * icon.
+	 * A themeIcon, Uri, or light and dark Uris to use as the terminal tab icon
```

Comment:
Since the type is clear we could avoid mentioning it in the comment:

```suggestion
	 * The icon for the terminal, used primarily in the terminal tab.
```